server/cmd/server/main.go

```diff
@@ -9,12 +9,15 @@ import (
 	"syscall"
 	"time"
 
+	"google.golang.org/grpc/credentials"
```

Comment:
goimports

Reply:
Indeed looks strange, but It is already formatted, that's how it looks.